.github/workflows/update.yaml

```diff
@@ -39,9 +39,9 @@ jobs:
             fi
 
             TAG_MINIMAL=$(cat ./$version/TAG_MINIMAL)
-            echo $LATEST_TAG > ./$version/TAG_MINIMAL
 
             if [[ "$TAG_MINIMAL" != "$LATEST_TAG" ]]; then
+              echo $LATEST_TAG > ./$version/TAG_MINIMAL
```

Comment:
I don't think this change alone will result in the expected behavior. This is because `TAG` files of all ubuntu versions are being updated when the `NEED_UPDATE` is set to 1:

https://github.com/cybozu/ubuntu-base/blob/4e8eab8f4a2f9a92dab69b38b78cf55c4350178f/.github/workflows/update.yaml#L87-L89

I think we need to modify this process.